api/tests/routes/backoffice_v3/offerers_test.py

```diff
@@ -910,12 +918,20 @@ def test_list_offerers_to_be_validated(self, authenticated_client):
 
             # when
             with assert_no_duplicated_queries():
-                response = authenticated_client.get(url_for("backoffice_v3_web.validation.list_offerers_to_validate"))
+                response = authenticated_client.get(
+                    url_for("backoffice_v3_web.validation.list_offerers_to_validate", order=order, sort=sort)
+                )
 
             # then
             assert response.status_code == 200
             rows = html_parser.extract_table_rows(response.data)
-            assert sorted([int(row["ID"]) for row in rows]) == sorted(o.id for o in to_be_validated_offerers)
+            # Without sort, table is ordered by dateCreated desc
+            to_be_validated_offerers.sort(
+                key=lambda x: getattr(x, sort) if sort else x.dateCreated, reverse=(order == "desc") if sort else True
```

Comment:
```suggestion
                key=attrgetter(sort or "dateCreated"), reverse=(order == "desc") if sort else True
```
avec `from operator import attrgetter`